src/tiledimage.js

```diff
@@ -1432,20 +1460,16 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
 
         // Load the new 'best' n tiles
         if (bestTiles && bestTiles.length > 0) {
-            bestTiles.forEach(function (tile) {
-                if (tile && !tile.context2D) {
+            for (let tile of bestTiles) {
+                if (tile) {
```

Comment:
We don't. `context2D` is just a proxy access to the cache, no longer a tile attribute. Checks on cache presence / validity are elsewhere. It just does not exist anymore.

Reply:
I think maybe you answered my question... I was asking how we make sure we don't try to load a tile that's already been loaded, and your answer is we do that elsewhere. I guess I'm curious where that elsewhere is.

Reply:
It is being done already in the current version of the master branch. Tile gets loaded by a new data only when it's key is not found in the cache, otherwise it refuses the cache object. Since c2d property newly just accesses the cache, it would be checking the same thing twice.

Reply:
Great, thank you for clarifying!